.env.example

```diff
@@ -39,4 +39,10 @@ DATAFILE_NAME = 'diabetes.csv'
 DB_CLUSTER_ID = ''
 
 # Optional. Container Image name for image creation
-IMAGE_NAME = 'mltrained'
\ No newline at end of file
+IMAGE_NAME = 'mltrained'
+
+# Run Evaluation Step in AML pipeline
+RUN_EVALUATION = 'true'
+
+# Set this flag to false from evaluation step to cancel pipeline run
```

Comment:
Set to true cancels the Azure ML pipeline run when evaluation criteria it not met.